app/models/application_record.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  MAX_TITLE_LENGTH = 50
```

Comment:
should these constants be in ApplicationRecord?

Reply:
They are used in several forms. How i can refactor this?

Reply:
let's move it to model (where it logically belongs). Book?

Reply:
![image](https://user-images.githubusercontent.com/38437488/63020132-ec02d280-bea5-11e9-94d1-06929724ecf3.png)